tests/MongoDB.Driver.Core.Tests/Core/Configuration/ConnectionSettingsTests.cs

```diff
@@ -94,7 +99,9 @@ public void constructor_with_applicationName_should_initialize_instance()
             var subject = new ConnectionSettings(applicationName: "app");
 
             subject.ApplicationName.Should().Be("app");
-            subject.Authenticators.Should().Equal(__defaults.Authenticators);
+            subject.AuthenticatorFactories.Should().NotBeNull();
+            var authenticators = subject.AuthenticatorFactories.Select(a => a.Create());
+            authenticators.Should().BeEmpty();
```

Comment:
Replace 102-104 with:

            subject.AuthenticatorFactories.Should().Equal(__defaults.AuthenticatorFactories);